package.json

```
@@ -1,6 +1,13 @@
 {
   "private": true,
-  "dependencies": {},
+  "scripts": {
+    "clean": "rm -rf dist && rm -rf .cache",
+    "dev": "bin/parcel serve index.html"
+  },
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "us-zcta-counties": "^0.0.2"
```

Can you move this to `devDependencies`? I don't have a strong understanding of dependencies vs devDependencies, but it looks like everything works even when everything is a devDependency, so I'm just trying to keep everything together for now. I'm also down to move everything to dependencies if that's better.

Also -- I've been using exact version numbers since we're not committing a lockfile. Maybe:

```suggestion
    "us-zcta-counties": "0.0.2"
```

---

Ah, it's needed at runtime and not just in development, so we should keep it in `dependencies`. I can update it to use the exact version :)